android/src/main/java/net/no_mad/tts/TextToSpeechModule.java

```diff
@@ -282,12 +284,54 @@ private boolean notReady(Promise promise) {
     }
 
     @SuppressWarnings("deprecation")
-    private int speak(String utterance, String utteranceId) {
+    private int speak(String utterance, String utteranceId, ReadableMap inputParams) {
+        int audioStreamType;
+
+        String audioStreamTypeString = inputParams.hasKey("KEY_PARAM_STREAM") ? inputParams.getString("KEY_PARAM_STREAM") : null;
+        float volume = inputParams.hasKey("KEY_PARAM_VOLUME") ? (float) inputParams.getDouble("KEY_PARAM_VOLUME") : (float) 1;
+        float pan = inputParams.hasKey("KEY_PARAM_PAN") ? (float) inputParams.getDouble("KEY_PARAM_PAN") : (float) 0;
```

Comment:
Wow! you made also `KEY_PARAM_VOLUME` and `KEY_PARAM_PAN`!!! Very nice! Thanks!!!

Reply:
I'm also very impressed... so considerate... 💯